prototypes/db/design.py

```diff
@@ -0,0 +1,354 @@
+import argparse
+import requests
+import logging
+import json
+import os
+from datetime import datetime
+from posixpath import join
+from pull import Pull
+
+#http://127.0.0.1:5984/_utils/#login - 000
+#http://10.30.133.78:5984/_utils/#login - djerba
+
+class Design():
+     ''' change location of database here with db and base'''
+     def __init__(self, db='djerba', base='http://admin:djerba123@10.30.133.78:5984/', level=logging.WARNING, log_path=None, filename=__name__):
+     #def __init__(self, db='test', base='http://admin:cgi@127.0.0.1:5984/', level=logging.WARNING, log_path=None, filename=__name__):
+          self.db = db
+          self.base = base
+          self.url = join(self.base,self.db)          
+          self.level = level
+          self.filename = filename
+          self.log_path = log_path
+          if log_path != None:
+               file_path = join(self.log_path, self.filename)
+               file_path += '.log'
+          else: file_path = None
+          logging.basicConfig(level=level, filename=file_path, format=f'%(asctime)s:%(filename)s:%(levelname)s: %(message)s', datefmt='%Y-%m-%d_%H:%M:%S')
+          
+     def CheckAndSwitchForwardSlashOrPlus(self, string_to_search, url=False):
```

Comment:
this method is also in pull.py, probably unnecessary to duplicate it, just call the one from pull.py no?